event.test.ts

```diff
@@ -278,6 +279,27 @@ describe('Event', () => {
 
       expect(isValid).toEqual(false)
     })
+
+    it('should return false for an invalid event id', () => {
+      const privateKey = 'd217c1ff2f8a65c3e3a1740db3b9f58b8c848bb45e26d00ed4714e4a0f4ceecf'
+
+      const { [verifiedSymbol]: _, ...event } = finishEvent(
+        {
+          kind: 1,
+          tags: [],
+          content: 'Hello, world!',
+          created_at: 1617932115,
+        },
+        privateKey,
+      )
+
+      // tamper with the id
+      event.id = event.id.replace(/0/g, '1')
```

Comment:
I updated it to replace the first 3 characters with 666 instead for maximum evil. Fortunately these tests are not random. The private key is hard-coded, so it will always produce the same result regardless.